static/app/components/modals/projectCreationModal.tsx

```diff
@@ -0,0 +1,264 @@
+            listClassName="centered"
+          />
+        </Fragment>
+      )}
+      {step === 1 && (
+        <Fragment>
+          <Subtitle>{t('Set your alert frequency')}</Subtitle>
+          <IssueAlertOptions
+            platformLanguage={platform?.language as SupportedLanguages}
+            onChange={updatedData => setAlertRuleConfig(updatedData)}
+          />
+          <Subtitle>{t('Name your project and assign it a team')}</Subtitle>
+          <ProjectNameTeamSection>
+            <div>
+              <Label>{t('Project name')}</Label>
+              <ProjectNameInputWrap>
+                <StyledPlatformIcon platform={platform?.key ?? 'other'} size={20} />
+                <ProjectNameInput
+                  type="text"
+                  name="project-name"
+                  placeholder={t('project-name')}
+                  autoComplete="off"
+                  value={projectName}
+                  onChange={e => setProjectName(slugify(e.target.value))}
+                />
+              </ProjectNameInputWrap>
+            </div>
+            <div>
+              <Label>{t('Team')}</Label>
+              <TeamInput
+                allowCreate
+                name="select-team"
+                aria-label={t('Select a Team')}
+                menuPlacement="auto"
+                clearable={false}
+                value={team}
+                placeholder={t('Select a Team')}
+                onChange={choice => setTeam(choice.value)}
+                teamFilter={(tm: Team) => tm.access.includes('team:admin')}
+              />
+            </div>
+          </ProjectNameTeamSection>
+        </Fragment>
+      )}
+      <Footer>
+        {step === 1 && <Button onClick={() => setStep(step - 1)}>{t('Back')}</Button>}
+        {step === 0 && (
+          <Button
+            priority="primary"
+            disabled={!platform}
+            onClick={() => setStep(step + 1)}
+          >
+            {t('Next Step')}
+          </Button>
+        )}
+        {step === 1 && (
+          <Button
+            priority="primary"
+            onClick={() => {
+              setCreating(true);
+              createProject();
+            }}
+            disabled={!projectName || !team || !alertRuleConfig || !platform || creating}
+          >
+            {t('Create Project')}
+          </Button>
+        )}
+      </Footer>
+    </Fragment>
+  );
+}
+
+const Footer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
+  gap: ${space(1)};
+  margin-top: ${space(2)};
+`;
+
+const StyledPlatformIcon = styled(PlatformIcon)`
+  position: absolute;
+  top: 50%;
+  left: ${p => p.theme.formPadding.md.paddingLeft}px;
+  transform: translateY(-50%);
+`;
+
+const ProjectNameInputWrap = styled('div')`
+  position: relative;
+`;
+const ProjectNameInput = styled(Input)`
+  padding-left: calc(${p => p.theme.formPadding.md.paddingLeft}px * 1.5 + 20px);
+`;
+
+export const modalCss = css`
+  width: 900px;
```

Comment:
max-width?
did you check this on mobile

Reply:
fixed - one bug with smaller screens and issue alert selection, but ill tackle that in a separate pr before we add this anywhere